server/test/test.spec.js

```diff
@@ -0,0 +1,24 @@
+/* global describe it */
+import chai from 'chai';
```

Comment:
Why is this file called test.spec when it is not the only test file? Name it according to what it is testing just like you did for the party test.